processing/src/main/java/org/apache/druid/query/groupby/epinephelinae/GroupByMergingQueryRunnerV2.java

```diff
@@ -122,11 +122,9 @@ public GroupByMergingQueryRunnerV2(
         CTX_KEY_MERGE_RUNNERS_USING_CHAINED_EXECUTION,
         false
     );
-    final QueryPlus<Row> queryPlusForRunners = queryPlus
-        .withQuery(
-            query.withOverriddenContext(ImmutableMap.of(CTX_KEY_MERGE_RUNNERS_USING_CHAINED_EXECUTION, true))
-        )
-        .withoutThreadUnsafeState();
+    final QueryPlus<Row> queryPlusForRunners = queryPlus.withQuery(
+        query.withOverriddenContext(ImmutableMap.of(CTX_KEY_MERGE_RUNNERS_USING_CHAINED_EXECUTION, true))
```

Comment:
Please extract ImmutableMap variable for better formatting

Reply:
done